google/auth/pluggable.py

```diff
@@ -163,6 +164,17 @@ def retrieve_subject_token(self, request):
                 "Executables need to be explicitly allowed (set GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES to '1') to run."
             )
         
+        # Check output file
+        if self._credential_source_executable_output_file is not None:
+            try:
+                output_file = open(self._credential_source_executable_output_file)
```

Comment:
prefer `with open(self._credential_source_executable_output_file) as output_file:` so the file can be closed automatically

Reply:
Good point 👍